--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,7 +27,7 @@ cleo = "^0.6.7"
 requests = "^2.18"
 cachy = "^0.2"
 requests-toolbelt = "^0.8.0"
-jsonschema = "^3.0a3"
+jsonschema = "^3.0.0a3"
 pyrsistent = "^0.14.2"
 pyparsing = "^2.2"
 cachecontrol = { version = "^0.12.4", extras = ["filecache"] }
--- a/setup.py
+++ b/setup.py
@@ -41,7 +41,7 @@ install_requires = \
  'cachy>=0.2,<0.3',
  'cleo>=0.6.7,<0.7.0',
  'html5lib>=1.0,<2.0',
- 'jsonschema>=3.0a3,<4.0',
+ 'jsonschema>=3.0.0a3,<4.0',
  'pkginfo>=1.4,<2.0',
  'pyparsing>=2.2,<3.0',
  'pyrsistent>=0.14.2,<0.15.0',
